packages/pods-core/src/index.ts

```diff
@@ -73,4 +73,13 @@ export function getAllImportPods(): PodClassEntryV4[] {
   ];
 }
 
+export enum CopyAsFormat {
+  "JSON" = "JSON",
+  "MARKDOWN" = "Markdown",
+}
+
+export function getAllCopyAsFormat(): CopyAsFormat[] {
```

Comment:
nit: since `CopyAsFormat` is a string enum we can grab all the values with `Object.values(CopyAsFormat)`